test/load/allocation/grpc/README.md

```diff
@@ -0,0 +1,80 @@
+# Load Test for gRPC allocation service
+
+This load tests aims to validate performance of the gRPC allocation service.
+
+## Kubernetes Cluster Setup
+
+For the load test you can follow the regular Kubernetes and Agones setup. In order to test the allocation performance in isolation, we let Agones to get all
+the game servers to Ready state before starting a test.
+Here are the few important things:
+- If you are running in GCP, use a regional cluster instead of a zonal cluster to ensure high availability of the cluster control plane
+- Use a dedicated node pool for the Agones controllers with multiple CPUs per node, e.g. `n1-standard-4'
+- In the default node pool (where the Game Server pods are created), make sure there are enough nodes available for all game servers to move into the ready state. We set the cluster count to 25 per zone (total of 75 per region)
```

Comment:
> (total of 75 per region)

Do you mean (total of 75 _across_ regions) ?

```suggestion
- In the default node pool (where the Game Server pods are created), 75 nodes are required to make sure there are enough nodes available for all game servers to move into the ready state. When using a regional cluster, with three zones with the region, that will require a configuration of 25 nodes per zone.
```

Is that better?

Reply:
SG. Updated.